app/models/petition.rb

```diff
@@ -28,19 +30,13 @@ class Petition < ActiveRecord::Base
 
   after_create :set_petition_on_creator_signature
 
-  searchable do
-    text :title
-    text :action
-    text :description
-    text :creator_name do
-      creator_signature.name
-    end
-    string :title
-    integer :signature_count
-    time :closed_at, :trie => true
-    time :created_at, :trie => true
-    string :state
-  end
+  extend Searchable(:title, :action, :description)
+  include Browseable
+
+  facet :all, -> { self.order(:signature_count).reverse_order }
+  facet :open, -> { self.for_state('open').order(:signature_count).reverse_order }
+  facet :closed, -> { self.for_state('closed').order(:signature_count).reverse_order }
+  facet :rejected, -> { self.for_state('rejected').order(:created_at).reverse_order }
```

Comment:
Should these be ``.reorder`` rather than ``.order``?  ``Browseable::Search#execute_search`` calls ``.reorder(:created_at)`` and then instance_execs the facet from params[:state] ontop, so we'd end up with ``ORDER BY created_at, signature_count desc`` right, when I think we'd expect ``ORDER BY signature_count desc``